lms/static/scripts/frontend_apps/services/grading.ts

```diff
@@ -6,6 +6,10 @@ export type Student = {
 
   /** API URL for posting outcome results. */
   LISOutcomeServiceUrl: string;
+
+  /** LTI user id for this student. */
+  lmsId: string;
```

Comment:
This was already part of the grading info we sent over the FE but it was not exposed/forwarded.